### DISCUSSION TITLE
NuxtJS Logo is not behaving correctly in github README

### DISCUSSION THREAD

**Discussion by seanghay** [nuxt/nuxt#18361] [General]
![image](https://user-images.githubusercontent.com/15277233/137014993-1ceb199f-78c1-422c-92c4-8f747064dd0f.png)

![image](https://user-images.githubusercontent.com/15277233/137015063-8b6669d5-886b-40d7-85b7-588d20e3f216.png)

## Solution 

To use stroke. For example, 

![image](https://camo.githubusercontent.com/76fc893540a16d0acb4967472a5195511ec64fd8d98f377cb00dc8fa73ffb67b/68747470733a2f2f7265666163746f72696e6775692e6e7963332e63646e2e6469676974616c6f6365616e7370616365732e636f6d2f7461696c77696e642d6c6f676f2d737469636b65722e737667)

**Comment by productdevbook** [comment]
<img width="755" alt="CleanShot 2021-10-12 at 22 22 06@2x" src="https://user-images.githubusercontent.com/38668796/137016419-6709c42e-f49d-4af2-83a1-0885ef5000fc.png">
I don't have a problem.

**Comment by seanghay** [comment]
> <img alt="CleanShot 2021-10-12 at 22 22 06@2x" width="755" src="https://user-images.githubusercontent.com/38668796/137016419-6709c42e-f49d-4af2-83a1-0885ef5000fc.png">
> 
> I don't have a problem.

Try changing your GitHub theme to light

**Comment by productdevbook** [comment]
I changed it, still no error.

**Comment by seanghay** [comment]
The problem occurs when System Theme doesn't match with GitHub Theme. The reason is that the SVG logo is using css to match with the System Theme. It happened in tailwind repo before.

**Comment by productdevbook** [comment]
> The problem occurs when System Theme doesn't match with GitHub Theme. The reason is that the SVG logo is using css to match with the System Theme. It happened in tailwind repo before.

<img width="800" alt="CleanShot 2021-10-12 at 22 43 00@2x" src="https://user-images.githubusercontent.com/38668796/137019063-90898cd1-4707-4872-bd18-31ae10877d5d.png">

Okay thats true

**Comment by seanghay** [comment]
<img src="https://user-images.githubusercontent.com/15277233/137019697-1ac72b9e-e328-4e60-a071-8a7e38d8da2f.png" width=300>

GitHub Mobile App

**Comment by productdevbook** [comment]
fix: [pr](https://github.com/nuxt/framework/pull/843)

**Comment by productdevbook** [comment]
please check -> https://github.com/productfrontenddeveloper/framework-1 and return feedback
